src/js/components/common/timeformat.js

```diff
@@ -0,0 +1,11 @@
+import moment from 'moment';
```

Comment:
The "react"/ idiomatic way would be to not rely on setting the locale internally in `moment`, but in the redux store instead and transform this into a "connected component" instead of a wrapper call to `moment` privates 😕. This way you would not only remove the reliance of some hidden global set somewhere in moment (which might be changed by a datepicker e.g.), but you would also gain the re-initialization of the user-defined locale on page load for free.
 (An additional benefit would be to later on getting rid of the `react-time` package, as this is unmaintained and a blocker for a pending react upgrade...)

BTW: the [proper approach to the formatting using moment](https://momentjs.com/docs/#/i18n/locale-data/) you must have surely missed during your intense study of the docs 😛.